src/types.jl

```diff
@@ -82,3 +82,6 @@ function Base.convert(::Type{T}, c::S) where {T<:AbstractSkyCoords,S<:AbstractSk
 end
 
 Base.:(==)(a::T, b::T) where {T<:AbstractSkyCoords} = lon(a) == lon(b) && lat(a) == lat(b)
+Base.isapprox(a::ICRSCoords, b::ICRSCoords; kwargs...) = isapprox(SVector(lon(a), lat(a)), SVector(lon(b), lat(b)); kwargs...)
+Base.isapprox(a::GalCoords, b::GalCoords; kwargs...) = isapprox(SVector(lon(a), lat(a)), SVector(lon(b), lat(b)); kwargs...)
+Base.isapprox(a::FK5Coords{e}, b::FK5Coords{e}; kwargs...) where {e} = isapprox(SVector(lon(a), lat(a)), SVector(lon(b), lat(b)); kwargs...)
```

Comment:
We can probably simplify this using a generic type
```suggestion
Base.isapprox(a::S, b::S; kwargs...) where {S <: AbstractSkyCoords} = isapprox(SVector(lon(a), lat(a)), SVector(lon(b), lat(b)); kwargs...)
```

Reply:
Ah that's a good point. We could add an additional generic version, but that's about as messy as just listing the multiple types.